docs/architecture/ADR-001-interest-stream.md

```diff
@@ -42,66 +38,57 @@ From [Cosmos IBC tutorial](https://tutorials.cosmos.network/understanding-ibc-de
 >
 > The only way to unlock the locked tokens on blockchain A is to send the voucher token back from blockchain B. The result is that the voucher token on blockchain B is burned. The burn process purposefully takes the tokens out of circulation.
 
-This means that if uTokens are to be sent to other Cosmos blockchains, then the interest stream must apply equally to 'voucher tokens' on other chains. These chains are not likely to be running our code, so it is unclear how we would cause uToken balances sent via IBC to generate interest.
-
-As an alternative, transfer of uTokens via IBC could be forbidden or unsupported - in that case, only the scenarios where uToken balances are held on Umee and Ethereum need to be addressed. This is highly undesirable.
+Behavior over IBC (and Ethereum) bridges can be expected to follow this pattern, where the _amount_ of uTokens transferred is represented faithfully, but any complex _behavior_ of uTokens is lost.
```

Comment:
```suggestion
This means that if uTokens are to be sent to other Cosmos blockchains, then the interest stream must apply equally to 'voucher tokens' on other chains. These chains are not likely to be running our code, so it is unclear how we would cause uToken balances sent via IBC to generate interest.

As an alternative, transfer of uTokens via IBC could be forbidden or unsupported - in that case, only the scenarios where uToken balances are held on Umee and Ethereum need to be addressed. This is highly undesirable.
```